lib/helper/Polly.js

```diff
@@ -168,13 +206,19 @@ class Polly extends Helper {
    */
   async stopMocking() {
     if (!this._checkIfMockingStarted()) return;
-    await this._disconnectPuppeteer();
 
-    const { polly } = this;
-    if (!polly) return;
-    await polly.flush();
-    await polly.stop();
-    delete this.polly;
+    if (this.currentDriver === 'Puppeteer') {
```

Comment:
I think you can make _disconnectPuppeteer and _disconnectWebDriver methods similarly to `_connect*`.

Reply:
yeah, that would be better